packages/example-vue/stories/Button.stories.js

```diff
@@ -3,13 +3,13 @@ import MyButton from './Button.vue';
 export default {
     title: 'Example/Button',
     component: MyButton,
-    argTypes: {
-        backgroundColor: { control: 'color' },
-        size: {
-            control: { type: 'select', options: ['small', 'medium', 'large'] },
-        },
-        onClick: {},
-    },
+    // argTypes: {
```

Comment:
Why is this code commented out?

Reply:
@eirslett - Apologies, I commented those out while testing to ensure the docgen info was generating the correct controls without the `argTypes` overriding it.

I'll fix it up now.